### PR TITLE
RFC: phy: restore operation at lower (50MHz) sys-clk-freq

### PR DESCRIPTION
**Pull request by gsomlo**
Following commit 15bc2db, LiteSDCard no longer works properly at the low end of system clock speeds (e.g., 50MHz). Reverting the way `self.ce` and `self.clk` are generated to `comb` instead of `sync`, allows LiteSDCard to operate at the full previous range of system clock speeds.

Fixes: 15bc2db

tested on nexys_video with vexriscv at 50 and 100 MHz, and with rocket at 50MHz on the nexys_video and ecpix5.

### CODE REVIEW

File: litesdcard/phy.py [hunk 1]
```diff
@@ -62,14 +62,14 @@ def __init__(self):
...
-        self.sync += self.ce.eq(clk & ~clk_d)
+        self.comb += self.ce.eq(clk & ~clk_d)
 
         # Ensure we don't get short pulses on the SDCard Clk.
         ce_delayed = Signal()
         ce_latched = Signal()
         self.sync += If(clk_d, ce_delayed.eq(self.clk_en))
         self.comb += If(clk_d, ce_latched.eq(self.clk_en)).Else(ce_latched.eq(ce_delayed))
-        self.sync += self.clk.eq(~clk & ce_latched)
+        self.comb += self.clk.eq(~clk & ce_latched)
```
Comment (gsomlo) [inline litesdcard/phy.py:72]: @enjoy-digital : I've understood and agree with all the *rest* of the changes in commits 1a3d474 and 15bc2db, but not sure why `self.ce` and `self.clk` were switched to `sync` from the original `comb`. It is the reason sdcard breaks at lower (50MHz) system clock frequencies.

To be honest, though, I also don't get why the change breaks sdcard at low speeds :)

**Comment by gsomlo** [comment]
@enjoy-digital : ping?

**Comment by enjoy-digital** [comment]
Hi @gsomlo,

sorry for the delay and thanks for investigating. The reason for the change was to improve the timings (https://github.com/enjoy-digital/litex/issues/1814). I'll have a closer look at this.

**Comment by enjoy-digital** [comment]
Thanks @gsomlo, that's merged. I'll take the time to understand. Just for info @Dolu1990, this will probably degrade the timings for your SoC -> I'll also redo some test and will try to find a proper fix.

**Comment by gsomlo** [comment]
> I'll also redo some test and will try to find a proper fix.

The weird thing is that if one switches `s/comb/sync/` only *one* of https://github.com/enjoy-digital/litesdcard/blob/master/litesdcard/phy.py#L65 or https://github.com/enjoy-digital/litesdcard/blob/master/litesdcard/phy.py#L65 sdcard still works at 50MHz sysclock. It only stops working if *both* of those lines are switched to `sync` !!!
